### DISCUSSION TITLE
Document

### DISCUSSION THREAD

**Discussion by giscus** [nvtaikma/Comment-blogs#4] [General]
# Document

http://127.0.0.1:5500/POST-DETAIL.html

**Comment by nvtaikma** [comment]
xin chao

**Reply by nvtaikma** [reply]
ok

**Comment by nvtaikma** [comment]
asd

**Comment by nvtaikma** [comment]
rat hay

**Comment by nvtaikma** [comment]
`<!DOCTYPE html>
<html>
<head>
<title>Page Title</title>
</head>
<body>

<h1>This is a Heading</h1>
<p>This is a paragraph.</p>

</body>
</html>`
